src/components/Error/Error.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import { NavLink } from 'react-router-dom'
+
+
+export interface ErrorProps {}
+
+const Error: React.FC<ErrorProps> = () => {
+    return (
+        <div className='error'>
+            <h2> Whoops! Looks like this path doesn't exist. Click the button to go back to the main page 🎨 </h2>
```

Comment:
Good call creating an error component - do we want to add an image as well for an added user experience?